app/models/health_care_application.rb

```diff
@@ -71,7 +71,7 @@ def send_failure_email?
   end
 
   def submit_sync
-    @parsed_form = override_parsed_form(parsed_form)
+    @parsed_form = HCA::OverridesParser.new(parsed_form).override
```

Comment:
This seemed a bit more logical than calling a method that calls this line of code.